pandas/tests/indexes/interval/test_interval.py

```diff
@@ -400,16 +400,9 @@ def test_get_item(self, closed):
                                              closed=closed)
         tm.assert_index_equal(result, expected)
 
-    # To be removed, replaced by test_interval_new.py (see #16316, #16386)
+    # To be removed after overlapping/non-monotonic/duplicate cases have been
+    # added to test_get_loc_scalar below
```

Comment:
The new `test_get_loc_scalar` only tests the monotonic non-overlapping case, so there are a few things in this test not fully covered.  Can add them to `test_get_loc_scalar` in this PR if desired, but didn't want to do so immediately since I didn't want to deviate too far from the existing tests.